OpenTabletDriver/Configurations/XP-Pen/Deco Fun XS.json

```
@@ -0,0 +1,31 @@
+{
+  "Name": "XP-Pen Deco Fun XS",
```

Change the name and update the filename to match.
```suggestion
  "Name": "XP-Pen CT430",
```